docs/tutorial/web-embeds.md

```diff
@@ -1,22 +1,50 @@
 # Web embeds in Electron
```

Comment:
Do we want to keep `In Electron` here? All our are features are technically "in Electron". 😅